src/commands/lambda/README.md

```diff
@@ -17,13 +17,13 @@ export AWS_SECRET_ACCESS_KEY="<ACCESS KEY>"
 datadog-ci lambda <command>
 ```
 
-### API
+Download the [Datadog CI][2].
 
-#### Configuration
+### Configuration
 
-Configuration is done via a json file, by default the tool load `datadog-ci.json` which can be overriden through the `--config` argument.
+Configuration is done using a JSON file. By default AWS Lambda loads `datadog-ci.json` which can be overriden through the `--config` argument.
```

Comment:
Can we add more specifics here - where is this JSON file - how do you access it - once you update it, do you need to do anything to refresh it?

Reply:
The original sentence was confusing, you need to specify the location of `datadog-ci.json` using the `--config` argument.